tests/test_flobject.py

```diff
@@ -943,3 +944,31 @@ def test_strings_with_allowed_values(load_static_mixer_settings_only):
         "density-based-implicit",
         "density-based-explicit",
     ]
+
+
```

Comment:
this test is taken from Adam's PR

Reply:
Yes, I checked this test locally and sufficient to test this integration.